client/state/analytics/reducer.js

```diff
@@ -3,11 +3,19 @@
  */
 import {
 	ANALYTICS_TRACKING_ON,
+	DESERIALIZE,
+	SERIALIZE,
 } from 'state/action-types';
 
 export const analyticsTracking = ( state = false, { type } ) => {
-	if ( ANALYTICS_TRACKING_ON === type ) {
-		return true;
+	switch ( type ) {
+		case ANALYTICS_TRACKING_ON:
+			return true;
+
+		// We don't want to persist across reloads
+		case SERIALIZE:
+		case DESERIALIZE:
+			return false;
 	}
```

Comment:
We should return the default value if you want to opt-out. If you use `createReducer` and don't pass in a schema, both functions will just return [intitialState]( https://github.com/Automattic/wp-calypso/blob/master/client/state/utils.js#L187)

You can also do this at a higher level, like we do for the [ui tree](https://github.com/Automattic/wp-calypso/blob/master/client/state/ui/reducer.js#L100) so any children will also not persist.

See also: https://github.com/Automattic/wp-calypso/blob/master/docs/our-approach-to-data.md#not-persisting-data